### DISCUSSION TITLE
Levels shown in random order

### DISCUSSION THREAD

**Discussion by Murpie007** [danthe1st/Mee6-bypasser#59] [Questions] [answered]
Hi, thanks for the awesome work. Your add on works like a charm. I have one question and I hope you can help. I made a list of about 20 roles.  When i do /show, it shows me the list in random order. Is there a way to show the list from level small to level high?

![levels](https://user-images.githubusercontent.com/103824020/163670873-437162bf-f09f-4457-abf3-e044215d4d3f.jpg)

**Comment by danthe1st** [comment, marked as answer]
Currently, there is no way to do this but it would be a simple thing to add.
If you want to do it by yourself (and have some programming knowledge), you can add a Pull Request.
If not, we might add this feature at some point.

**Reply by Murpie007** [reply]
Thank you for your reply, Sir.  I have no programming knowledge, so I'm afraid I have no clue how to add this.
Guess I'll just have to wait patiently until the feature is added.

**Reply by danthe1st** [reply]
I have added this. The output of `/show` should now be sorted by level.

**Reply by Murpie007** [reply]
It does! You are awesome, Sir! Thank you so much 🙏🏻
